test/unit/mem/view/src/ViewPlainPtrTest.cpp

```diff
@@ -109,7 +110,9 @@ TEMPLATE_LIST_TEST_CASE("viewPlainPtrOperatorTest", "[memView]", alpaka::test::T
     alpaka::test::testViewPlainPtrOperators<TestType, float>();
 }
 
-TEMPLATE_TEST_CASE("createView", "[memView]", (std::array<float, 4>), std::vector<float>)
+using CreateViewTestTypes = std::tuple<std::array<float, 4>, std::vector<float>>;
```

Comment:
Will this fixed with C++20?

Reply:
I do not know, maybe.

Reply:
Yes, it is fixed in C++20. I added an comment.